packages/components/image/src/image.ts

```diff
@@ -52,9 +52,12 @@ export const imageProps = buildProps({
     default: () => mutable([] as const),
   },
   /**
-   * @description whether to append image-viewer to body. A nested parent element attribute transform should have this attribute set to `true`.
+   * @description whether to append image-viewer to body. A nested parent element attribute transform should have this attribute set to `true`. Defaults to true in table.
    */
-  previewTeleported: Boolean,
+  previewTeleported: {
+    type: definePropType<boolean | undefined>([Boolean]),
```

Comment:
```suggestion
    type: Boolean,
```
Declaring the type directly should be enough.